test/next-13.5/next-13.5.test.ts

```diff
@@ -0,0 +1,23 @@
+// TODO: Officially support Next 13
+// Validate our types are exported correctly:
+import type { createDynamicRouteParser } from "../../dynamic-routes";
+import type { createDynamicRouteParser as _ } from "../../dynamic-routes/next-13";
+import type { MemoryRouterProvider } from "../../MemoryRouterProvider";
+import type { MemoryRouterProvider as __ } from "../../MemoryRouterProvider/next-13";
```

Comment:
We'll need to create another entry-point here too:

```suggestion
import type { MemoryRouterProvider as __ } from "../../MemoryRouterProvider/next-13.5";
```